### ISSUE TITLE
to_pandas NA bug

### ISSUE THREAD

**Issue by vojtechmusilek** [rick-jennings/phable#5]
phable: 0.1.12
exception: 'NA' object has no attribute 'unit'
how to replicate:
```py
from phable.client import Client
with Client(uri, username, password) as client:
  res = client.eval('[{ts: now(), val: 0}, {ts: now()+1s, val: na()}].toGrid.addMeta({id: @test})')
  df = res.to_pandas()
```

**Comment by vojtechmusilek** [comment]
From pandas probably [pandas.NA](https://pandas.pydata.org/docs/user_guide/missing_data.html#missing-data-na) or from numpy `numpy.nan`

**Comment by rick-jennings** [comment]
That's what I was thinking.

I just created a branch for this issue:
https://github.com/rick-jennings/phable/tree/add_pd_na_support

From this branch I also released to PyPI Phable version 0.1.13a1

Would you please try out Phable version 0.1.13a1?

I believe you would need to install it like this:
```
pip install phable==0.1.13a1
```

**Comment by vojtechmusilek** [comment]
Thanks for patch, it works.
